SwiftCompilerSources/CMakeLists.txt

```diff
@@ -251,6 +251,12 @@ else()
 
 #include \"Parse/RegexParserBridging.h\"
 ")
+  add_custom_command(
+    OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp"
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different
+      "${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp.tmp"
+      "${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp"
+  )
```

Comment:
This is avoid mtime of `HeaderDependencies.cpp` being updated every time `cmake` configuration happens.